subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheProblemReportingIntegrationTest.groovy

```diff
@@ -30,6 +30,41 @@ import static org.gradle.integtests.fixtures.configurationcache.ConfigurationCac
 @IgnoreIf({ GradleContextualExecuter.isNoDaemon() })
 class ConfigurationCacheProblemReportingIntegrationTest extends AbstractConfigurationCacheIntegrationTest {
 
+    def "input files are reported with relative path"() {
+        given:
+        file('provider.txt').text = 'provider'
+        file('fis-path.txt').text = 'fis-path'
+        file('fis-file.txt').text = 'fis-file'
+        buildFile '''
+            providers.fileContents(layout.projectDirectory.file("provider.txt")).with { provider ->
+                println("provider = ${provider.asText.get()}")
+            }
+
+            new FileInputStream("fis-path.txt").withCloseable { fis ->
```

Comment:
IMO, it would be nice to have `FileInputStream(file("...."))` (i.e. with absolute path as an argument) covered there as well. It is somewhat covered in other tests but this one looks more focused.

Reply:
Good point. Done.